{
  "type": "issue",
  "issue": {
    "id": 2582320476,
    "node_id": "I_kwDOABII586Z6xVc",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31075",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31075/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31075/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31075/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/31075",
    "number": 31075,
    "state": "open",
    "state_reason": null,
    "title": "Please....Big Noob here..(this right place?)",
    "body": "### Is there an existing issue for this?\n\n- [X] I have searched the existing issues\n\n### Current behaviour\n\n Hello, Big noob here. I appreciate you taking the time to read this. I have tried scouring the internet including Github for issues similar to what I am experiencing while Syncing The blockchain on my Host Machine. This is not my first time syncing Core to my Pc, but it is the first time I'm experiencing an issue. The same issue arises both on my Linux OS (same machine) and on my Windows OS. Somewhere in the early stages of syncing (Progress %3.68) i get an error called : System error while flushing: Fatal LevelDB error: Corruption: Block checksum mismatch: C:\\users\\amazi\\appdata\\local\\bitcoin\\chainstate\\001325.ldb\n\n### Expected behaviour\n\nI expected to be able to to sync the blockchain without any errors, especially early in. it doesn't happen right away but it does happen.\n\n### Steps to reproduce\n\nI just downloaded the Bitcoin Core 28.0 Version and verified Via Gpg keys for matching hash, then proceeded to sync with bitcoin-qt GUI on windows.\n\n### Relevant log output\n\n2024-10-11T23:13:01Z UpdateTip: new best=00000000000000003563a19a8a0d96a2ca0bf6bb7b4e4ac9df94eab68ba22b56 height=304711 version=0x00000002 log2_work=79.042624 tx=40297150 date='2014-06-07T21:28:52Z' progress=0.036862 cache=716.4MiB(5836705txo)\r\n2024-10-11T23:13:03Z Fatal LevelDB error: Corruption: block checksum mismatch: C:\\Users\\amazi\\AppData\\Local\\Bitcoin\\chainstate/001325.ldb\r\n2024-10-11T23:13:03Z You can use -debug=leveldb to get more complete diagnostic messages\r\n2024-10-11T23:13:03Z [error] A fatal internal error occurred, see debug.log for details: System error while flushing: Fatal LevelDB error: Corruption: block checksum mismatch: C:\\Users\\amazi\\AppData\\Local\\Bitcoin\\chainstate/001325.ldb\r\n\n\n### How did you obtain Bitcoin Core\n\nCompiled from source\n\n### What version of Bitcoin Core are you using?\n\n28.0\n\n### Operating system and version\n\nwindows 11, 64bit\n\n### Machine specifications\n\n11th gen Intel core i7-11700k @ 3.60 ghz\r\n32 gig ram \r\n\r\ndx12\r\n\r\n3070ti \r\n\r\nwindows 11, 64 bit",
    "user": {
      "login": "DpTheSaint",
      "id": 128269078,
      "node_id": "U_kgDOB6U7Fg",
      "avatar_url": "https://avatars.githubusercontent.com/u/128269078?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/DpTheSaint",
      "html_url": "https://github.com/DpTheSaint",
      "followers_url": "https://api.github.com/users/DpTheSaint/followers",
      "following_url": "https://api.github.com/users/DpTheSaint/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/DpTheSaint/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/DpTheSaint/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/DpTheSaint/subscriptions",
      "organizations_url": "https://api.github.com/users/DpTheSaint/orgs",
      "repos_url": "https://api.github.com/users/DpTheSaint/repos",
      "events_url": "https://api.github.com/users/DpTheSaint/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/DpTheSaint/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": false,
    "comments": 1,
    "created_at": "2024-10-11T23:21:49Z",
    "updated_at": "2024-10-11T23:29:43Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 2408235250,
      "node_id": "IC_kwDOABII586PisDy",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/2408235250",
      "actor": {
        "login": "DpTheSaint",
        "id": 128269078,
        "node_id": "U_kgDOB6U7Fg",
        "avatar_url": "https://avatars.githubusercontent.com/u/128269078?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/DpTheSaint",
        "html_url": "https://github.com/DpTheSaint",
        "followers_url": "https://api.github.com/users/DpTheSaint/followers",
        "following_url": "https://api.github.com/users/DpTheSaint/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/DpTheSaint/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/DpTheSaint/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/DpTheSaint/subscriptions",
        "organizations_url": "https://api.github.com/users/DpTheSaint/orgs",
        "repos_url": "https://api.github.com/users/DpTheSaint/repos",
        "events_url": "https://api.github.com/users/DpTheSaint/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/DpTheSaint/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2024-10-11T23:29:42Z",
      "updated_at": "2024-10-11T23:29:42Z",
      "author_association": "NONE",
      "body": "i got another popup when i tried to exit QT.\r\n\r\nMicrosoft Visual C++ Runtime Library:\r\n\r\nAssertion failed!\r\nProgram c:\\Program Files\\Bitcoin\\bitcoin-qt.exe\r\nfile: txdb.cpp\r\nLine: 105\r\n\r\nexpression: old_heads[0] == hashBlock\r\n\r\nfor information on how your program can cause an assertion failure, see the visual c++ docs\r\n\r\nnot sure if this is relevant but thought i'd share it . it popped up after the original error pop-up.",
      "user": {
        "login": "DpTheSaint",
        "id": 128269078,
        "node_id": "U_kgDOB6U7Fg",
        "avatar_url": "https://avatars.githubusercontent.com/u/128269078?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/DpTheSaint",
        "html_url": "https://github.com/DpTheSaint",
        "followers_url": "https://api.github.com/users/DpTheSaint/followers",
        "following_url": "https://api.github.com/users/DpTheSaint/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/DpTheSaint/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/DpTheSaint/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/DpTheSaint/subscriptions",
        "organizations_url": "https://api.github.com/users/DpTheSaint/orgs",
        "repos_url": "https://api.github.com/users/DpTheSaint/repos",
        "events_url": "https://api.github.com/users/DpTheSaint/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/DpTheSaint/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/31075#issuecomment-2408235250",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31075"
    }
  ]
}